pkg/security/agent/agent.go

```diff
@@ -100,7 +100,7 @@ func (rsa *RuntimeSecurityAgent) StartEventListener() {
 			if err == io.EOF || in == nil {
 				break
 			}
-			log.Infof("Got message from rule `%s` for event `%s` with tags `%+v` ", in.RuleID, string(in.Data), in.Tags)
+			log.Tracef("Got message from rule `%s` for event `%s` with tags `%+v` ", in.RuleID, string(in.Data), in.Tags)
```

Comment:
Hey Nick,
This PR has been delayed for a bit, but I'm working on it again. Although I understand why this is helpful for writing rules, the problem is that logging this line has a significant performance impact (and unfortunately, we run the agent with debug level activated). We have a plan to improve logging in an upcoming card, hopefully this will help :)